packages/wallet/src/redux/reducer.ts

```diff
@@ -36,6 +37,13 @@ export function initializedReducer(
   state: states.Initialized,
   action: actions.WalletAction,
 ): states.WalletState {
+  if (createsNewProcess(action)) {
```

Comment:
It would be good to start stubbing out tests for these. Even if there's not much to test yet.

Reply:
I actually tried to! I still don't feel like I know how to mock things :/

I will try again.

Reply:
I'll leave the tests out of this PR -- later, I will actually call out to the protocol reducers, and will stub those out.